rails 3.1 support

vincenzodev opened the pull request.

Commit:
Added compatibility with Rails 3.1

sentience commented:
This pull request is obsolete and should be closed. The gemspec now specifies Rails 3.0.0 or newer:

```
s.add_dependency "rails", ["> 3.0.0"]
```

vincenzodev committed:
Fixed problem with has_enum attributes with the same name of different models
Merge branch 'master' of https://github.com/openteam/has_enum

Conflicts:
	has_enum.gemspec

vincenzodev committed:
lapse
